src/sentry/static/sentry/app/components/gridEditable/styles.tsx

```diff
@@ -28,10 +25,42 @@ type GridEditableProps = {
   isDragging?: boolean;
 };
 
-export const GridPanel = styled(Panel)`
-  /* overflow: hidden; */
+export const Header = styled('div')`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  margin-bottom: ${space(1)};
+`;
+export const HeaderTitle = styled('h2')`
+  margin: 0;
+
+  font-size: 20px;
```

Comment:
Is there a size in `theme`?

Reply:
It's `18px` and `22px` in the theme file but `20px` when guesstimating from the Figma design. I've changed to use `theme` and will let @doralchan tweak it.